backend-api/src/config/user_db.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@darkwhitenez good call to sperate production database, make sure you store whatever attribute you have in the .env file

you can also use URL format to reduce env overhead:
`mysql://USERNAME:PASSWORD@HOST:PORT/DB`